doc/general/operation-modes/operation-modes.md

```diff
@@ -34,6 +34,38 @@ The queue nutshell serves to encapsulate the functionality of a receipt chain fo
 
 The Helper nutshell is used to support updates and configurations.
 
+#### fiskaltrust.SignatureCloud
+
+The fiskaltrust.SignatureCloud is a pure online solution. The receipt linking is handled entirely online. The advantage of this solution is that no installation or configuration is required for the client and any platform can use this service.
+
+The fiskaltrust.SignatureCloud has two different versions:
+ - 1.1
+ - 1.2
+
+We differentiate between them by backend servers. So the external url is the same: https://signaturcloud.fiskaltrust.at for both versions. If you want to use a specific version of fiskaltrust.SignatureCloud, you will need to set the service-version parameter into the header. As default the fiskaltrust.SignatureCloud 1.1 version is used.
```

Comment:
in future updates we need to clarify market-related top-level-domain.

Reply:
what do you think moving this section into appendix-at-rksv/operation-modes/operation-modes.md file can be a solution for this market-related top-level-domain issue?